Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -295,6 +294,31 @@ def IccLeftChart (x y : ℝ) [h : Fact (x < y)] :
     have B : Continuous fun z : EuclideanSpace ℝ (Fin 1) => z 0 := continuous_apply 0
     exact (A.comp B).comp continuous_subtype_val
 
+variable {x y : ℝ} [hxy : Fact (x < y)]
+
+-- TODO: how to avoid passing hxy_twice?
```

Comment:
I'd appreciate help with this question: I'm not sure how to solve this. Removing `hxy_twice`, even if writing `include hxy in` before the theorem, yields an error "failed to synthesize Fact (x ≤ y)".

The same issue appears three more times below.

Reply:
@YaelDillies I know you're not a differential geometer, but since you're looking right now: do you happen to know the answer? This is just a Lean question, I presume.

Reply:
Maybe an instance `Fact (a < b) -> Fact (a <= b)` is sensible

Reply:
Btw please include more context by making a multiline comment next time. It's really painful to navigate from this comment to the relevant piece of code on mobile

Reply:
Sorry, I'll try to include more context next time. Thanks for the tip, the solution worked (I made it a scoped instance).